.github/workflows/linters.yml

```diff
@@ -25,11 +25,11 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
-          node-version: "12.x"
+          node-version: "18.x"
```

Comment:
- Kindly check your lighthouse linters, actually they are not passing GitHub checks. You can check the guide provided by Microverse [HERE](https://github.com/microverseinc/linters-config/tree/master/html-css)